app/models/spree/subscription_frequency.rb

```diff
@@ -8,8 +8,11 @@ class SubscriptionFrequency < Spree::Base
     validates :title, :months_count, presence: true
     with_options allow_blank: true do
       validates :months_count, numericality: { greater_than: 0, only_integer: true }
-      validates :title, uniqueness: { case_sensitive: false }
     end
+    validates :months_count, numericality: { greater_than: 0, only_integer: true, less_than_or_equal_to: 12 }, allow_blank: true
 
+    def display_frequency
+      "#{months_count} #{title}"
```

Comment:
what if months_count is blank?

Reply:
it cant be blank please check above it validates presence

Reply:
i had checked thats why i wrote that comment :)

`with_options allow_blank: true` see its present :)

Reply:
how can be select tag field empty? we have done modification in gem. Check below